iwyu.cc

```diff
@@ -2549,6 +2549,19 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
     return true;
   }
 
+  // The C++ spec does not allow us to apply typeid to an incomplete type.
+  // Therefore, we must report the type to be included.
+  bool VisitCXXTypeidExpr(clang::CXXTypeidExpr* expr) {
+    if (CanIgnoreCurrentASTNode()) return true;
```

Comment:
Would be nice if you could clang-format the new code, the `return true;` should be on a new line. I did a mass cleanup earlier this year: https://github.com/include-what-you-use/include-what-you-use/commit/890a3150d74f834a31e57c649195d6d68da682b1.